core/kazoo_endpoint/src/kz_endpoint.erl

```diff
@@ -1119,6 +1119,12 @@ maybe_get_t38(Endpoint, Call) ->
                                                        )
     end.
 
+-spec maybe_rtcp_mux(boolean()) -> kz_json:object().
+maybe_rtcp_mux('false') ->
+    [{<<"RTCP-MUX">>, false}];
+maybe_rtcp_mux(_) ->
+    [].
+
```

Comment:
this should change to
```
-spec maybe_rtcp_mux(ccv_acc()) -> ccv_acc().
maybe_rtcp_mux({Endpoint, Call, CallFwd, CCVs} = Acc) ->
    case kz_json:get_value([<<"media">>, <<"rtcp_mux">>], Endpoint) of
        'undefined' -> Acc;
        Value -> {Endpoint, Call, CallFwd, kz_json:set_value(<<"RTCP-MUX">>, kz_term:is_true(Value), CCVs)}
    end.
```

and change CCVFuns in `generate_ccvs/3` to include this
```
generate_ccvs(Endpoint, Call, CallFwd) ->
    CCVFuns = [fun maybe_retain_caller_id/1
...
              ,fun maybe_set_webrtc/1
              ,fun maybe_rtcp_mux/1
              ],